python/tk_framework_desktopserver/server.py

```diff
@@ -36,12 +38,37 @@ def __init__(self, port=None, debug=True, whitelist=None, keys_path=None):
         self._whitelist = whitelist or self._DEFAULT_WHITELIST
         self._debug = debug
 
-        self._logger_root = logging.getLogger("tk-framework-desktopserver")
+        self._logger_root = get_logger()
```

Comment:
I feel that the previous form of using `logging.getLogger()` is much clearer. This explicitly tells the reader of the code that this method comes from the logging module. By just saying `get_logger()`, it's not clear if this is an internal method or some external thing.